packages/shared/styles/components/molecules/SfBanner.scss

```
@@ -1,81 +1,93 @@
   box-sizing: border-box;
-  display: flex;
-  min-height: var(--banner-min-height, 9rem);
-  padding: var(--banner-padding, var(--spacer-big));
-  color: var(--banner-color, var(--c-text));
-  background-repeat: no-repeat;
+  min-height: var(--banner-min-height, 14rem);
+  width: var(--banner-width, 100%);
   background-position: var(--banner-background-position, bottom left);
-  background-size: cover;
-  background-color: var(--banner-background);
+  background-size: var(--banner-background-size, cover);
+  background-color: var(--background-color);
   background-image: var(--background-image);
+  background-repeat: var(--banner-background-repeat, no-repeat);
+  color: var(--banner-color, var(--c-text));
+  letter-spacing: var(--banner-letter-spacing, 0.1px);
   &__container {
-    flex: var(--banner-container-flex, 0 0 100%);
+    display: flex;
+    justify-content: left;
+    flex-direction: column;
+    margin: var(
+      --banner-container-margin,
+      var(--spacer-xl) 0 0 var(--spacer-xl)
+    );
   }
   &__subtitle {
-    margin: var(--banner-subtitle-margin);
-    color: var(--banner-subtitle-color, var(--c-dark-variant));
-    font: var(
+    margin: var(--banner-subtitle-margin, 0);
+    color: var(
+      --banner-color,
+      var(--banner-subtitle-color, var(--c-dark-variant))
+    );
+    @include font(
       --banner-subtitle-font,
-      font(
-        --banner-subtitle,
-        300,
-        var(--font-size-regular),
-        1.16,
-        var(--font-family-primary)
-      )
+      var(--font-normal),
+      var(--font-2xs),
+      1.4,
+      var(--font-family-secondary)
     );
+    text-transform: var(--banner-subtitle-text-transform, uppercase);
   }
   &__title {
-    margin: var(--banner-title-margin);
-    color: var(--banner-title-color);
-    font: var(
+    margin: var(--banner-title-margin, 0.6875rem 0 0 0);
+    color: var(--banner-color, var(--banner-title-color, var(--c-text)));
+    @include font(
       --banner-title-font,
-      font(
-        --banner-title,
-        300,
-        var(--font-size-extra-big),
-        1.6,
-        var(--font-family-secondary)
-      )
+      var(--font-normal),
+      var(--h2-font-size),
+      1.4,
+      var(--font-family-secondary)
     );
+    text-transform: var(--banner-title-text-transform, uppercase);
   }
   &__description {
-    display: var(--banner-description-display, none);
-    margin: var(--banner-description-margin);
-    font: var(
+    display: var(--banner-display-description, none);
+    margin: var(
+      --banner-description-margin,
+      var(--spacer-sm) 0 var(--spacer-base) 0
+    );
+    color: var(--banner-color, var(--banner-description-color, var(--c-text)));
+    @include font(
       --banner-description-font,
-      font(
-        --banner-description,
-        300,
-        var(--font-size-big),
-        1.6,
-        var(--font-family-secondary)
-      )
+      var(--font-light),
+      var(--font-base),
+      1.7,
```

```suggestion
      1.6,
```
try use 1.4 or 1.6 line-height :) is the best sizes for line-height

---

this line-height is often different on designs, I found 1.2, 1.8, even 1.9. Should I choose always one of these two: 1.4 or 1.6?